doc/source/examples/03_modeling/scale_map_bodies.mystnb

```diff
@@ -0,0 +1,233 @@
+from ansys.geometry.core.sketch.sketch import Sketch
+from ansys.geometry.core.math.frame import Frame
+from ansys.geometry.core.math.vector import UnitVector3D
+from ansys.geometry.core.math.plane import Plane
+```
+
+```{code-cell} ipython3
+# Initialize the modeler for this example notebook
+m = Modeler()
+```
+
+## Scale Body
+
+The `scale` function is designed to modify the size of 3D bodies by a specified scale factor. This function is a important part of geometric transformations, allowing for the dynamic resizing of bodies.
+
+### Usage
+
+To use the `scale` function, you call it on an instance of a geometry body, passing a single argument: the scale value. This value is a real number (`Real`) that determines the factor by which the body's size will be changed.
+
+```python
+body.scale(value)
+
++++
+
+## Example: Making a Cube
+The code snippets below demonstrates how to change the size of a cube using the `scale` function in `body.py`. The process involves initializing a sketch design for the cube, defining the shape parameters, and then performing rescaling operation to generate the new shape.
+1. **Initialize the Cube Sketch Design:**
+   - A new design sketch named "cube" is created.
+
+```{code-cell} ipython3
+design = m.create_design("cube")
+```
+
+2. **Define Cube Parameters:**
+   - `side_length` is set to 10 units, representing the side length of the cube.
+
+```{code-cell} ipython3
+# Cube parameters
+side_length = 10
+```
+
+3. **Create the Profile Cube:**
+   - A square box is created centered on the origin using `side_length` as the side length of the square.
+
+```{code-cell} ipython3
+# Square with side length 10
+box_sketch = Sketch().box(Point2D([0, 0]), side_length, side_length)
+```
+
+4. **Create Cube Body:**
+   - `extrude_sketch` on the `box_sketch`, as the base sketch, creates the 3D cube with `distance` being `side_length`.
+
+```{code-cell} ipython3
+# Extrude the cube profile by a distance of side_length
+cube = design.extrude_sketch("box", box_sketch, side_length)
+
+design.plot()
+```
+
+5. **Making the cube twice as large:**
+   - Copy the original cube. Using `scale` with a value of 2, we will double the side lengths of the cube, thereby making the body twice as large, and offset it to view the difference.
+
+```{code-cell} ipython3
+# Copy the original cube
+doubled = cube.copy(cube.parent_component, "doubled_box")
+# Double the size
+doubled.scale(2)
+# Translate the copied cube in the x direction
+doubled.translate(UnitVector3D([1, 0, 0]), 30)
+
+design.plot()
+```
+
+6. **Halfing the size of the *original* cube:**
+   - Copy the original cube. By using `scale` with a value of 0.5, we will effectively halve the side lengths of the cube, and then offset it to view the difference.
+   - Note: Since we doubled the size of the cube in the previous cell, we need to use the 0.25 factor which in turn will translate to half the size of the original cube.
```

Comment:
```suggestion
   - Note: Because the size of the cube in the previous cell was doubled, using the 0.25 factor translates it to half the size of the original cube.
```